addons/test_mail/models/mail_test_access.py

```diff
@@ -88,3 +88,13 @@ def _get_customer_information(self):
             if not values.get('phone'):
                 values['phone'] = record.mobile
         return email_key_to_values
+
+
+class MailTestAccessAdmin(models.Model):
```

Comment:
Use ``mail.test.access`` with access being admin, should do the work and avoid yet another test model.